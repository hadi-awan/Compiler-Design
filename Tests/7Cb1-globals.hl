// Test Global variable declarations and initializations
var a, B, #c;
a;
B;

// Test assignments
a = 10;
a;
B = {1,2,3,4,5};
B;

// Test assignment and use of Boolean identifiers
#c = #1;
if #c then println "#c is true"; else println "#c is false"; fi;

// Test retrieval and use of Number identifiers
var d, e;
d = -2; 
e = (a+d)*a/d;
println "e is: ",e;