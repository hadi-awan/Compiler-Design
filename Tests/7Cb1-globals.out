0
{}
10
{1, 2, 3, 4, 5}
#c is true
e is: -40
